### PR TITLE
Added email notification for probation period ends.

### PR DESCRIPTION
**Pull request by vovikboyko**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Lunaflumen — APPROVED** [review]

**Review by sooooolo — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/EmailSending/NewPlastMemberEmailGreeting.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        {
+            try
+            {
+                (await GetNewPlastMembersAsync()).ToList().ForEach(async (user) => await SendEmailGreetingForNewPlastMemberAsync(user.Email));
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+            return true;
```
Comment (voroniak) [inline EPlast/EPlast.BLL/Services/EmailSending/NewPlastMemberEmailGreeting.cs:37]:
```suggestion
        public async Task NotifyNewPlastMembersAsync()
        {
                (await GetNewPlastMembersAsync()).ToList().ForEach(async (user) => await SendEmailGreetingForNewPlastMemberAsync(user.Email));

```
Could it be like this ?

**Comment by voroniak** [comment]
Could you also fix 2 Code Smells and add some unt tests ?

File: EPlast/EPlast.BLL/Services/EmailSending/NewPlastMemberEmailGreeting.cs (hunk 1 above)
Comment (vovikboyko) [reply EPlast/EPlast.BLL/Services/EmailSending/NewPlastMemberEmailGreeting.cs:37]: Yes, corrected

**Comment by vovikboyko** [comment]
> Could you also fix 2 Code Smells and add some unt tests ?

Of course)

**Review by Lunaflumen — APPROVED** [review]

**Review by randomVovaKuhot — APPROVED** [review]

**Pull request merged**